### ISSUE TITLE
Dotnet-sdk package is very out of date

### ISSUE THREAD

**Issue by faldor20** [NixOS/nixpkgs#73193]
**Describe the bug**
The dotnet-sdk package is still running version 2.0 when 3.0 is available as a stable release

**To Reproduce**
1. ... install package
2. ... look at the version of dotnet-sdk installed

**Expected behavior**
The dotnet-sdk package version should be at least version 3.0

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: dotnet-sdk
```

**Comment by faldor20** [comment]
fixing the issue should be as simple as having some change this https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/dotnet/sdk/default.nix to have a newer version.
It shows the maintainer as @kuznero .

Thanks

**Comment by kuznero** [comment]
@faldor20, will try to have a look shortly

**Comment by kuznero** [comment]
Should be addressed with #72300

**Comment by jonringer** [comment]
dotnet-sdk 2.1,3.0,3.1 are no available after #73262
